Core.ApplicationServices/SystemUsage/Migration/ItSystemUsageMigrationService.cs

```diff
@@ -146,69 +145,68 @@ public class ItSystemUsageMigrationService : IItSystemUsageMigrationService
                 return OperationFailure.Forbidden;
             }
 
-            using (var transaction = _transactionManager.Begin())
+            using var transaction = _transactionManager.Begin();
```

Comment:
@a-naskret this "huge" change is just the removal of the `{}` from the using block